app/views/providers/application_merits_task/has_other_involved_children/show.html.erb

```diff
@@ -0,0 +1,51 @@
+<%= page_template page_title: t('.page_title', count: "#{pluralize(@legal_aid_application.involved_children.count, 'child')}"), template: :basic do %>
+
+  <% if @legal_aid_application.involved_children.count > 0 %>
+    <%= govuk_fieldset_header(size: 'l') { t('.existing', count: "#{pluralize(@legal_aid_application.involved_children.count, 'child')}")} %>
+    <div class="govuk-summary-list">
+      <table class="govuk-table">
+        <thead class="goveuk-table__head">
```

Comment:
random e here 'gov**e**uk-table__head' instead of 'govuk-table__head'

Reply:
Eagle-eyed!  👍